### ISSUE TITLE
False positive for apt-get install --assume-yes (DL3014)

### ISSUE THREAD

**Issue by chauncey-garrett** [hadolint/hadolint#259]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
The error would not be thrown.
### Actual behavior
An error is thrown when `--assume-yes` is used; one is NOT thrown when `-y` is used instead.
### Steps to reproduce the behavior

```bash
apt-get --no-install-recommends --assume-yes install build-essential
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 1.11.1-no-git
```

**Comment by lorenzo** [comment]
Thanks, we'll look into this

**Comment by lorenzo** [comment]
@chauncey-garrett Can you tell us the error code that should be fixed?
